### ISSUE TITLE
Conflict between GeoJSON and Prezi 4 term `transform` ?

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#2302]
At least need to use a scoped context.

Check that this is legit? (there's `transformation` for geo-referencing) Could be invalid?

**Comment by tomcrane** [comment]
Yes it's fine! They will never conflict if new `@context` for navPlace value.
